plugins/unmanaged_files/unmanaged_files_inspector.rb

```diff
@@ -53,6 +53,17 @@ def extract_rpm_database
         files[pair.first] = pair[1]
       end
     end
+    # make sure that all parent directories of managed rpm directories are considered
+    # managed
+    dirh.dup.each do |d, e|
```

Comment:
Unused block argument - `e`. If it's necessary, use `_` or `_e` as an argument name to indicate that it won't be used.